locale/en-us/setting.lua

```diff
@@ -211,37 +211,37 @@ config.IntelliSense.traceBeSetted        =
 'Please read [wiki](https://github.com/sumneko/lua-language-server/wiki/IntelliSense-optional-features) to learn more.'
 config.IntelliSense.traceFieldInject     =
 'Please read [wiki](https://github.com/sumneko/lua-language-server/wiki/IntelliSense-optional-features) to learn more.'
-config.diagnostics['unused-local']          = -- TODO: need translate!
-'未使用的局部变量'
-config.diagnostics['unused-function']       = -- TODO: need translate!
-'未使用的函数'
-config.diagnostics['undefined-global']      = -- TODO: need translate!
-'未定义的全局变量'
-config.diagnostics['global-in-nil-env']     = -- TODO: need translate!
-'不能使用全局变量（ `_ENV` 被设置为了 `nil`）'
-config.diagnostics['unused-label']          = -- TODO: need translate!
-'未使用的标签'
-config.diagnostics['unused-vararg']         = -- TODO: need translate!
-'未使用的不定参数'
-config.diagnostics['trailing-space']        = -- TODO: need translate!
-'后置空格'
-config.diagnostics['redefined-local']       = -- TODO: need translate!
-'重复定义的局部变量'
-config.diagnostics['newline-call']          = -- TODO: need translate!
-'以 `(` 开始的新行，在语法上被解析为了上一行的函数调用'
+config.diagnostics['unused-local']          =
+'Enable unused local variable diagnostics.'
+config.diagnostics['unused-function']       =
+'Enable unused function diagnostics.'
+config.diagnostics['undefined-global']      =
+'Enable undefined global variable diagnostics.'
+config.diagnostics['global-in-nil-env']     =
+'Enable cannot use global variables （ `_ENV` is set to `nil`） diagnostics.'
+config.diagnostics['unused-label']          =
+'Enable unused label diagnostics.'
+config.diagnostics['unused-vararg']         =
+'Enable unused vararg diagnostics.'
+config.diagnostics['trailing-space']        =
+'Enable trailing space diagnostics.'
+config.diagnostics['redefined-local']       =
+'Enable redefined local variable diagnostics.'
+config.diagnostics['newline-call']          =
+'Enable newline call diagnostics. Is\'s raised when a line starting with `(` is encountered, which is syntactically parsed as a function call on the previous line.'
 config.diagnostics['newfield-call']         = -- TODO: need translate!
 '在字面量表中，2行代码之间缺少分隔符，在语法上被解析为了一次索引操作'
```

Comment:
I honestly don't know what this one does. If you give me an example of when this one is triggered, I can also add this one.

Reply:
> I honestly don't know what this one does. If you give me an example of when this one is triggered, I can also add this one.

```lua
local t = {
    require
    'name'
}
```